awx/ui_next/src/screens/Host/Host.jsx

```diff
@@ -117,62 +196,99 @@ class Host extends Component {
         </PageSection>
       );
     }
-
     return (
-      <PageSection>
-        <Card className="awx-c-card">
-          {cardHeader}
-          <Switch>
-            <Redirect from="/hosts/:id" to="/hosts/:id/details" exact />
-            {host && (
-              <Route
-                path="/hosts/:id/edit"
-                render={() => <HostEdit match={match} host={host} />}
-              />
-            )}
-            {host && (
-              <Route
-                path="/hosts/:id/details"
-                render={() => <HostDetail match={match} host={host} />}
-              />
-            )}
-            {host && (
-              <Route
-                path="/hosts/:id/facts"
-                render={() => <HostFacts host={host} />}
-              />
-            )}
-            {host && (
-              <Route
-                path="/hosts/:id/groups"
-                render={() => <HostGroups host={host} />}
-              />
-            )}
-            {host && (
+      <>
+        <PageSection
```

Comment:
Things got complicated with the PageSections in Inventory and Host.  This css prop eliminates extra padding if the user is coming from Inventories